download.html

```diff
@@ -20,6 +20,17 @@
 <div class="container">
     <div class="download_main">
 
+    <h2>Installing with go-xcat <sup class="new">New!</sup> </h2>
+
+    To easily install xCAT and handle the dependency package, a new script <b>go-xcat</b> is provided.
+    <br>
+    <br>On the target xCAT Management Node, run the following commands to install the latest version of xCAT:
+    <pre class="codetext">
+       wget http://xcat.org/files/go-xcat -O - &gt;/tmp/go-xcat
+       chmod +x go-xcat
+       ./go-xcat
```

Comment:
If the file was download to `/tmp/go-xcat` in line 29, line 30 and 31 should use `/tmp/go-xcat` instead of `go-xcat` as well.